account_payment_order/tests/test_payment_order_outbound.py

```diff
@@ -48,6 +48,7 @@ def _create_supplier_invoice(self):
                 "payment_mode_id": self.env.ref(
                     "account_payment_mode.payment_mode_outbound_ct1"
                 ).id,
+                "invoice_date": datetime.today(),
```

Comment:
It's better to have a specific date. Why do you want to have it today? In any case, you should use `fields.Datet.today()` instead.

Reply:
Ok, thanks @pedrobaeza 
I changed it to use `fields.Date.today()` instead. I opted for that as that was the default date applied by Odoo before its removal: https://github.com/odoo/odoo/blob/fcaa54939e9a4f0dd5e47cd0ccffe7aa24bd451c/addons/account/models/account_move.py#L111
So, for minimal impact in the actual result, I think it would be better to use that.